expo-test.js

```diff
@@ -0,0 +1,70 @@
+const { spawn } = require("child_process")
```

Comment:
Is there any risk with not using `cross-spawn` for Windows systems?

Reply:
Good question, I don't know to be honest. I'll see if I can find out.

Reply:
🤷 otherwise, I don't see any issues.